xml/admin_ceph_upgrade.xml

```diff
@@ -129,35 +129,37 @@
       <para>
        Run the <command>salt-run upgrade.check</command> command to verify that
        the cluster uses the new bucket type <literal>straw2</literal> and that
-       the &adm; is not a storage node. The default is <literal>straw2</literal>
-       for any newly created buckets.
+       the &adm; is not a storage node. The default is
+       <literal>straw2</literal> for any newly created buckets.
       </para>
       <important>
-        <para>
-          The new <literal>straw2</literal> bucket type fixes several limitations
-          in the original <literal>straw</literal> bucket type. The previous
-          <literal>straw</literal> buckets would change some mappings that should
-          have changed when a weight was adjusted. <literal>straw2</literal> achieves
-          the original goal of only changing mappings to or from the bucket item
-          whose weight has changed.
-        </para>
-        <para>
-          Changing a bucket type from <literal>straw</literal> to <literal>straw2</literal> results
-          in a small amount of data movement, depending on how much the bucket
-          item weights vary from each other. When the weights are all the same, no data
-          will move. When an item's weight varies significantly there will be more movement.
-          To migrate, execute:
-        </para>
+       <para>
+        The new <literal>straw2</literal> bucket type fixes several limitations
+        in the original <literal>straw</literal> bucket type. The previous
+        <literal>straw</literal> buckets would change some mappings that should
+        have changed when a weight was adjusted. <literal>straw2</literal>
+        achieves the original goal of only changing mappings to or from the
+        bucket item whose weight has changed.
```

Comment:
I am not sure, but is there a missing "not" in here?

Reply:
this way is correct (but vague) - taken from upstream https://docs.ceph.com/en/latest/rados/operations/crush-map/#hammer-crush-v4